script.js

```diff
@@ -66,128 +65,151 @@ searchField.addEventListener("keyup", (e) => {
+    backButton = document.createElement("button");
+    backButton.id = "back-button";
+    backButton.innerText = "Back to Shows";
+    document.body.appendChild(backButton);
+    header.appendChild(backButton);
+  } else {
+    backButton = document.createElement("button");
+    backButton.id = "back-button";
+    backButton.innerText = "backButton";
+    document.body.appendChild(backButton);
+    document.body.removeChild(backButton);
+  }
+  const button = document.getElementById("back-button");
+  button.addEventListener("click", () => {
+    return location.reload();
+  });
+}
+
 function makePageForEpisodes(episodesObject) {
   rootElem.className = "root";
-  //console.log(episodesObject.url)
   episodesObject.forEach((episode) => {
-    //console.log(episode.url);
-    //console.log(episode.url.includes(".com/shows"));
-    
     const episodesContainer = document.createElement("div");
     rootElem.appendChild(episodesContainer);
     const episodeName = document.createElement("h1");
     episodeName.className = "episode-name";
     episodesContainer.className = "episode-container";
     episodesContainer.id = episode.id;
+
+    
+    const infoContainer = document.createElement("div");
+    infoContainer.className = 'info-container';
+    const rating = document.createElement("p");
+    rating.innerHTML = `<b>Rated:</b> ${episode.rating.average}`
+    if(episode.rating.average) {
+      infoContainer.appendChild(rating);
+      }
+    const genres = document.createElement("p");
+    genres.innerHTML = `<b>Genres:</b> ${episode.genres}`;
+    if(episode.genres) {
+      infoContainer.appendChild(genres);
+    }    
+    const status = document.createElement('p');
+    status.innerHTML = `<b>Status:</b> ${episode.status}`;
+    if(episode.status) {
+      infoContainer.appendChild(status);
+    }    
+    const runtime = document.createElement('p');
+    runtime.innerHTML = `<b>Runtime:</b> ${episode.runtime}`;
+    if(episode.runtime) {
+      infoContainer.appendChild(runtime);
+    }    
+    
+    episodesContainer.addEventListener("click", episodeContainerEvent);
+
     if (episode.season && episode.number) {
       episodeName.innerText = `${episode.name} - ${episodeCode(
         episode.season,
-        episode.number
+        episode.number        
       )}`;
+        
     } else episodeName.innerText = episode.name;
     episodesContainer.appendChild(episodeName);
+    episodesContainer.appendChild(infoContainer);
     const episodeImage = document.createElement("img");
-
     if (!episode.image) {
       episodeImage.src = "";
     } else {
       episodeImage.src = episode.image.medium;
     }
-
     episodesContainer.appendChild(episodeImage);
     const episodeSummary = document.createElement("p");
-    episodeSummary.className = "episode-text"
+    episodeSummary.className = "episode-text";
+
     episodeSummary.innerHTML = episode.summary;
     episodesContainer.appendChild(episodeSummary);
+    //truncated text
```

Comment:
Truncating long texts and implementing read more / read less functionality is a great idea. The below commented out code is a good start and I encourage you to finish it.